src/main/java/io/muehlbachler/bswe3b/CalculatorService.java

```diff
@@ -2,12 +2,20 @@
 
 import org.springframework.stereotype.Service;
 
-// FIXME: add javadoc
+/**
+ * Represents a calculator service, which provides only division so far.
+ */
 @Service
 public class CalculatorService {
-    // FIXME: add javadoc
+    /**
+     * Divides two int type numbers and returns the ans.
+     *
+     * @param a The dividend.
+     * @param b The divisor.
+     * @return The ans of the division.
+     */
     public int divide(final int a, final int b) {
-        // FIXME: implement
-        return -1;
+        // if only division by zero is the problem, then this should be b == 0
```

Comment:
this documentation is not necessary here; it's pretty clear from the method documentation and the code itself.